mavlink/qgroundcontrol
# Make scrolling wheel on actuator test not be a significant increment

Description
-----------
When you use a mouse and scroll up to give just a single input, the power given to the actuators is incredibly big (see screenshot).
![image](https://github.com/user-attachments/assets/b1c03e15-4f47-4e5b-acd1-7c2e082b0799)

To prevent such unexpected large jumps in the amount of power given to the actuators, this PR limits how much the value actually jumps to be 1% per scroll input.

Test Steps
-----------
Connect with a drone (preferribly a SITL such as gazebo), go to vehicle setup, then actuators and enable the sliders. Then, simply scroll up and down with your mouse over the sliders.

Checklist:
----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
